Client/Telemetry/GleanWrapper.swift

```diff
@@ -17,7 +18,11 @@ struct DefaultGleanWrapper: GleanWrapper {
     func handleDeeplinkUrl(url: URL) {
         Glean.shared.handleCustomUrl(url: url)
     }
-
+    
```

Comment:
We should remove trailing spaces as well. We use an Xcode option to help us have that automatically. Those are the options I have in my Xcode:
![Screenshot 2023-06-08 at 2 34 32 PM](https://github.com/mozilla-mobile/firefox-ios/assets/11338480/91fb85dd-1928-40f4-b0b2-860d7ed1aa21)
![Screenshot 2023-06-08 at 2 34 43 PM](https://github.com/mozilla-mobile/firefox-ios/assets/11338480/dc289a9d-a34f-42fc-9003-5c9b43b9c40c)

```suggestion

```